### PR TITLE
Prerelease cleanup

### PR DESCRIPTION
**Pull request by johnmay**
Still one more change set I'm waiting to submit. But in the meantime here is some clean up prior to release. The README has bugged me for a long time, now much cleaner. Could take a back up of the current one on the Wiki if you like.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,241 +3,65 @@
...
-
-**PLEASE NOTE**: This is a library of useful data structures and algorithms to manipulate them 
-from the area of structural chemo- and bioinformatics. As such, it is intended for the use by
-programmers, who wish to save some effort by reusing code. It is not intended for the enduser. 
-If you consider yourself to be more like user, you might not find what you wanted. 
-Please refer to other projects like the JChemPaint project (http://jchempaint.github.com/)
-or the Jmol project (http://www.jmol.org/) for programs that actually take advantage of the 
-CDK library.
-
-## Compiling
```
Comment (egonw) [inline README.md]: Looks good, but I really like to to have build instructions as part of the repo. Really, wikis get lost all the time, and people downloading an archive should not have to rely on other information to figure out how to build it. I know all those install info is trivial now, but it won't be in a few years.

Please consider moving what you removed from the README.md into an INSTALL.md file.

Comment (johnmay) [reply README.md]: Build instructions are still there.

Comment (johnmay) [reply README.md]: Under "INSTALL", could also add an INSTALL.md that goes into more details.

Comment (johnmay) [reply README.md]: This link is easiest to read: https://github.com/cdk/cdk/blob/ad91090df348c858532e70e6677e6d4d200d876b/README.md.

The issue in the past was tests used to fail so we had to have a lot of explanation about skipping tests etc. Now you just type ``mvn install`` and if it doesn't work something is wrong.

Comment (egonw) [reply README.md]: Yes, that's true. But I was thinking indeed about the less ideal situation... where things don't build well. They do now (have you already tried Java11, btw?), but those tricks of how to compile/install/test stuff when the situation is no longer ideal (likely in a few years; or different platform), that is still relevant (as it has been for many years). This is the developer talking that has seen many perfect solutions break down over the years, and people struggling with new build systems :)

So, an INSTALL.md with some of the original tips, hints, etc would still be welcome. I'll merge in this one now.

**Pull request merged**